app/src/main/java/kth/jjve/xfran/repositories/ResultRepo.java

```diff
@@ -1,22 +1,39 @@
 package kth.jjve.xfran.repositories;
 
 
+import android.util.Log;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.firestore.DocumentReference;
+import com.google.firebase.firestore.FirebaseFirestore;
+
 import androidx.lifecycle.MutableLiveData;
 
-import java.io.IOException;
+import java.time.LocalDate;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 import kth.jjve.xfran.models.Result;
+import kth.jjve.xfran.models.Workout;
 
 public class ResultRepo {
 
     private static final String LOG_TAG = ResultRepo.class.getSimpleName();
+
     private static ResultRepo instance;
+    FirebaseAuth firebaseAuth;
+    FirebaseFirestore firebaseFirestore;
+    String userID;
+    String resultID;
+    private Result result = new Result();
+    private MutableLiveData<Result> up;
```

Comment:
up was meant to be for user profile (right @jitseve ?) so I would refactor the variable to sth else

Reply:
So that's what up stands for!! I was like "ok that's an odd name that jitse chose" hahaha.
I figured it wouldnt be an issue cos it's private, but changed it to make more sense (now that i know what up stands for)